CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## Release v0.9.7
```

Comment:
This should be `v0.9.8` now?

Reply:
Yeah, sorry. Noticed it too late. I'll fix it on master,.